lms/djangoapps/courseware/tests/test_course_info.py

```diff
@@ -13,6 +14,10 @@
 
 from lms.djangoapps.ccx.tests.factories import CcxFactory
 from openedx.core.djangoapps.self_paced.models import SelfPacedConfiguration
+from openedx.core.djangoapps.site_configuration.tests.test_util import with_site_configuration_context
+from openedx.core.djangoapps.waffle_utils.testutils import WAFFLE_TABLES
+from openedx.features.enterprise_support.tests.mixins.enterprise import EnterpriseTestConsentRequired
+from pyquery import PyQuery as pq
```

Comment:
nit: duplicate of above [#R13](https://github.com/open-craft/edx-platform/pull/98/files#diff-b1a8cfd5f056c92a4ca1bdbb076dfe62R13)